.github/workflows/pull-mirrors-from-db.yml

```
@@ -22,45 +22,14 @@ jobs:
-              *)
-                TEXT="Very old images"
-                ;;
-            esac
-            echo "   | $TEXT | \`rsync -av rsync://fi.mirror.armbian.de/$storage\` | $size |" >> tmp.md
-          done
-          sed -n '/^## Current Mirrors/,$p' "$FILE" >> tmp.md
-          sed -i -e :a -e '/^\n*$/{$d;N;ba' -e '}' tmp.md
-          mv tmp.md "$FILE"
-  
       - name: Update Current Mirrors section
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:41:17: Double quote to prevent globbing and word splitting

(shellcheck)